c2corg_ui/static/js/addassociation.js

```diff
@@ -83,11 +83,12 @@ app.AddAssociationController = function(appApi, appDocument) {
  */
 app.AddAssociationController.prototype.associate = function(doc) {
   var parentId, childId;
+  var type = this.parentDoctype;
```

Comment:
`type === 'images'` is needed, but if you ask about `var type = this.parentDoctype;` it's not, I just found it shorter to write 3 times 'type' rather than 'this.parentDoctype'.